_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -5,11 +5,6 @@ rule_type: atomic
 description: |
   This rule checks that each `object` element  has a non-empty accessible name.
 accessibility_requirements:
-  wcag20:1.1.1: # Non-text Content (A)
-    forConformance: true
-    failed: not satisfied
-    passed: further testing needed
-    inapplicable: further testing needed
```

Comment:
Instead of removing the mapping, I think we should fix that sometimes this rule is failing the wrong SC. I think the simplest way to do this is to either add an assumption, or exclude objects with certain types of content.